data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1020 @@
+			choice
+				`	"We'd better get started then."`
+					goto embark
+				`	"Can you tell me more about the Kijra-pets?"`
+					goto kijra-pet
+				`	"How does Sioeora even know that the Kijra-pets are on this moon?"`
+					goto knowledge
+				`	"Are you certain you're okay going with me?"`
+					goto youokay
+				`	"Is there anything I should watch out for?"`
+					goto watchout
+			label jumpdrives
+			`	"Remember, Captain, our Predecessors did ply the stars for ten thousand years before we last set eyes on thy own race, who by then had only just discovered industry. Drives like the one within thy ship have been known to us for a long, long time."`
+			choice
+				`	"We'd better get going."`
+					goto embark
+				`	"Do the scions have their own jump drives?"`
+					goto warning
+				`	"What more can you tell me about the Kijra-pets?"`
+					goto kijra-pet
+				`	"Are you certain you're okay going with me?"`
+					goto youokay
+				`	"Is there anything I should watch out for?"`
+					goto watchout
+			label warning
+			`	"That is not for me to know, nor, I think, for thee."`
+			choice
+				`	"We'd better get going."`
+					goto embark
+				`	"Are you certain you're okay going with me?"`
+					goto youokay
+				`	"Is there anything I should watch out for?"`
+					goto watchout
+			label embark
+			`	"I agree," Sieasej says.`
+				accept
+	on stopover
+		conversation
+			`Shortly after you enter the system, Sieasej enters your ship's cockpit and directs you to a location in the planet's southern hemisphere, on the edge of one of the smaller continents. "Here, captain," they say, "'twas a location for one of the old breeding grounds. When last our ships visited here, 'twas this location 'twere they noted most of the Kijran."`
+			`	As you approach the coordinates to land, you see that it contains the remains of a substantial complex; over a dozen large buildings and several disused landing pads are present, connected by paths of smooth, untarnished metal. The materials show a curious quality of being both unblemished and overgrown; vines cling to otherwise-spotless walls and the various walkways, though often shaded by large plants, do not show any debris or detritus on their surfaces.`
+			branch retrieve-tranqs
+				or
+					has "outfit: Enforcer Riot Staff"
+					has "outfit: Pug Peacekeeping Staff"
+			`	You set the <ship> down on one of the landing pads and exit onto the surface, though not before retrieving a net from storage. You are accompanied by Sieasej, who holds aloft a cylindrical device which they wave in the air for several seconds. "I read several calls of the species," they say. "We are fortunate, captain. We will not have to travel far." They wave their device around some more, making slower but wider motions, before they eventually speak again. "The nearest one lies to the west, about a hundred meters."`
```

Comment:
What are "calls" supposed to mean here? Do you mean the actual noises they make? If so, "read" is a very confusing verb, although you might need to do a bit of restructuring to fit a better one.